images/clusterloader2/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.14.4 AS clusterloader-builder
+RUN go version
+WORKDIR /go/src/k8s.io
+RUN git clone https://github.com/njtran/perf-tests
```

Comment:
Can we just build upstream, or did you have custom changes you needed to make?

Reply:
I have it like this to test custom changes not necessary for this PR. I'll change it back to upstream.